src/Nancy/ViewEngines/ViewNotFoundException.cs

```diff
@@ -29,14 +30,14 @@ public ViewNotFoundException(string viewName, string[] availableViewEngineExtens
             this.AvailableViewEngineExtensions = availableViewEngineExtensions;
             this.InspectedLocations = inspectedLocations;
 
-            this.message = String.Format(
-                    "Unable to locate view '{0}'{4}Currently available view engine extensions: {1}{4}Locations inspected: {2}{4}Root path: {3}{4}" +
-                    "If you were expecting raw data back, make sure you set the 'Accept'-header of the request to correct format, for example 'application/json'",
-                    this.ViewName,
-                    string.Join(",", this.AvailableViewEngineExtensions),
-                    string.Join(",", this.InspectedLocations),
-                    this.rootPathProvider.GetRootPath(),
-                    Environment.NewLine);
+            this.message = string.Format(

```

Comment:
`StringBuilder`? Or do we care? There's a ton of `string.Join`, `string.Concat` and `string.Format` here...